### DISCUSSION TITLE
api setting the group

### DISCUSSION THREAD

**Discussion by ghost** [drakkan/sftpgo#1529] [Q&A] [answered]
when creating a user with the api can someone show me an example of how to set the primary group?

**Comment by vepauk** [comment, marked as answer]
Hi,

Here is the sample json payload for User creation:

{
	"username": "sample",
	"password": "Sample123!",
    "home_dir": "E:\\sample\\%username%",
    "permissions": {"/": ["list","download"]},
    "status": 1,
    "groups": [
            {
                "name": "Sample Group",
                **_"type": 1_**
            }
        ]
	}
	
	For Primary, type = 1
	For Secondary, type = 2
	For memberhsiop type = 3

**Comment by ghost** [comment]
thank you!
